src/multi_sio.c

```diff
@@ -11,7 +11,7 @@ u32 gMultiSioIntrFuncBuf[0x120 / 4] = {}; // Interrupt Routine RAM Execution Buf
 
 struct MultiSioArea gMultiSioArea = {};
 
-#if (CURRENT_GAME <= GAME_SA2)
+#if (GAME <= GAME_SA2)
```

Comment:
I'm actually not sure if this counts as game or engine 😭

Reply:
Yeah, it's a bit blurry here